jdaviz/configs/mosviz/helper.py

```diff
@@ -5,7 +5,70 @@
 
 class MosViz(ConfigHelper):
     """MosViz Helper class"""
-    _default_configuration = 'mosviz'
+
+    _default_configuration = "mosviz"
+
+    def load_data(
+        self,
+        onedspectra,
```

Comment:
`onedspectra` is a bit of word jumble and bit hard to parse? Maybe `spectra_1d`, or `oned_spectra`?